scripts/wasmd/deploy_erc20.js

```diff
@@ -137,7 +137,7 @@ async function main() {
   const wallet = await Secp256k1HdWallet.fromMnemonic(alice.mnemonic);
   const client = new SigningCosmWasmClient(httpUrl, alice.address0, wallet);
 
-  const wasm = fs.readFileSync(__dirname + "/contracts/cw-erc20.wasm");
+  const wasm = fs.readFileSync(__dirname + "/contracts/cw_erc20.wasm");
```

Comment:
Why change the filename?

Reply:
The older rust-optimizer always created a `contract.wasm`, which we then manually renamed. The new rust-optimizer uses the output names from Rust directly to allow writing multiple contracts in one out directory. So next time the nameservices is recompiled, it gets the underscore too.

See also https://github.com/CosmWasm/cosmwasm-examples/commit/a13efad8e9c4a8fd009291f13b1f6912464fd26d